### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by lucafranco** [Kevin-Mattheus-Moerman/Abaqus-Installation-Instructions-for-Ubuntu#2]
The installation process stops because of the following error:

```
An error occurred while executing the action.

Technical details:
Creating child process failed. Log ID is 0001
/home/luca/Documents/PC_Utilities/Abaqus/4/SIMULIA_fesafe/Linux64/1/inst/common/installationFiles/code/unzip -qo "/home/luca/Documents/PC_Utilities/Abaqus/4/SIMULIA_fesafe/Linux64/1/inst/common/installationFiles/code/siteIDandDslsStat.zip" -d "/tmp/DSY_20190325T225926+0100/tmp"
Stdout: 
Stderr: unzip: loadlocale.c:130: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))' failed.

Action LaunchAppAction from feature CODE\linux_a64\SMATocLicPanel failed.
Action ID: unzip_siteIDandDslsStat
```

What can be the solution to this? Thanks in advance

**Comment by Kevin-Mattheus-Moerman** [comment]
Sorry I do not know I'm afraid. I started this repo to document my own steps to installing Abaqus and to share it with others. I am however not an expert in trouble shooting messages like this. If you do find a solution please post it here to and we'll add it to the instructions.

**Comment by lucafranco** [comment]
I'm struggling for days but nowadays I don't have a solution :/ if I find I'll surely share it. Ps. It's Abaqus 2019 on Elements OS Juno

**Comment by beeb** [comment]
@lucafranco still no luck? I have the same error

**Comment by lucafranco** [comment]
Nope. I haven't tried since then, I've given up. Do you know someone who
could help us?

Il giorno mar 16 lug 2019 alle ore 16:12 Valentin Bersier <
notifications@github.com> ha scritto:

**Comment by beeb** [comment]
I ended up skipping all the license server setup steps and it finished installing successfully. Then I manually added the license server in a config file somewhere. Not at my computer right now but I can find the info later

**Comment by lucafranco** [comment]
And it works? Yes, if you please can send me the which files of the license
server you placed manually on the installation folder, and the path on your
computer where u placed them I would be grateful, thank you.

Luca

**Comment by beeb** [comment]
The file is under `/var/DassaultSystemes/Licenses/DSLicSrv.txt`. Its contents is simply the hostname of your license server including port.
e.g. `my.license.server:4085`

**Comment by beeb** [comment]
You can check that the path to this text file is correct in `/usr/DassaultSystemes/SimulationServices/V6R2018x/linux_a64/SMA/site/custom_v6.env`:

    license_server_type=DSLS
    dsls_license_config="/var/DassaultSystemes/Licenses/DSLicSrv.txt"

**Comment by marandra** [comment]
@lucafranco @beeb The assert fails due to an issue related to locales (not exactly sure how). Try prefixing LC_ALL=C when executing the setup script:
`$ LC_ALL=C csh path/to/setup`

**Comment by NMudunuru** [comment]
> @lucafranco @beeb The assert fails due to an issue related to locales (not exactly sure how). Try prefixing LC_ALL=C when executing the setup script:
> `$ LC_ALL=C csh path/to/setup`

This worked for me! Thanks @marandra

**Comment by lucafranco** [comment]
The solution by @marandra worked for me too, now I have another problem that I suppose is not related to this. Thanks a lot!

**Comment by TorquatoGarulli** [comment]
> @lucafranco @beeb The assert fails due to an issue related to locales (not exactly sure how). Try prefixing LC_ALL=C when executing the setup script:
> `$ LC_ALL=C csh path/to/setup`

Worked for me too ;)
